test/query.jl

```diff
@@ -286,16 +286,20 @@ end
 
 end
 @testset "Multiple Magic bytes" begin
-    q = query(joinpath(file_dir, "magic1.tiff"))
-    @test typeof(q) == File{format"TIFF"}
-    q = query(joinpath(file_dir, "magic2.tiff"))
-    @test typeof(q) == File{format"TIFF"}
+    q = query(joinpath(file_dir, "ffc.pcx"))
+    @test typeof(q) == File{format"PCX"}
     open(q) do io
         @test position(io) == 0
         skipmagic(io)
-        @test position(io) == 4
```

Comment:
Where would it be best to put these `skipmagic` definitions? In `registry.jl` or `query.jl` where skipmagic is defined.